kt/godot-library/src/main/kotlin/godot/core/Variant.kt

```diff
@@ -745,7 +745,8 @@ enum class VariantType(
 
     internal val toGodot = { buffer: ByteBuffer, any: Any? ->
         if (any == null) {
-            NIL.toGodotWithoutNullCheck(buffer, Unit)
+            // TODO: replace with NIL.toGodotWithoutNullCheck(buffer, Unit) once https://youtrack.jetbrains.com/issue/KT-68339 is fixed!
```

Comment:
Can we create an issue for this one so we don't forget ?

Reply:
done: https://github.com/utopia-rise/godot-kotlin-jvm/issues/643